src/ngraph/op/quantize.hpp

```diff
@@ -0,0 +1,60 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/axis_set.hpp"
+#include "ngraph/op/op.hpp"
+#include "ngraph/type/element_type.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        class Quantize : public ngraph::op::Op
+        {
+        public:
+            enum class RoundMode
+            {
+                HALF_AWAY_FROM_ZERO,
+                HALF_TO_EVEN
+            };
+
+            Quantize(std::shared_ptr<Node> input,
+                     std::shared_ptr<Node> scale,
```

Comment:
With the scale, offset design is there a way the backend would know which MODE is used to calculate the scale. The reason I am asking is because we should be able to distinguish between SCALED/MIN_COMBINED/MIN_FIRST mode for the transformer to make a decision on calling mkldnn or eigen

Reply:
I designed this API specifically so that nGraph would not have to know about MODE.  The modes you mentioned come from TensorFlow, but there may me more as we expand to other frameworks.  We need to have a discussion on how to integrate MKLDNN kernels into this schema.  I talked with @jbobba and he indicated it was possible though I am short on the exact details.

Reply:
Offline discussion:  MKLDNN cares about having the offset=0.  Hence we are covered.